### PR TITLE
Added View to get monthly cost of the Redshift cluster divided to schema

### PR DESCRIPTION
**Pull request by kerbelp**
After the admin will set the cost of the cluster and the amount of MB in his cluster, he could query the view to know how the cost is divided through all the schemas

### CODE REVIEW

File: src/AdminViews/v_cost_per_schema.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+(
+  dbase_name,
+  schemaname,
+  usage_megabytes,
+  usage_percentage_with_free_space,
+  monthly_usage_cost
+)
+AS
+ SELECT t.dbase_name, t.schemaname, sum(t.megabytes) AS usage_megabytes, sum(t.totalsum_actual_usage_percentage) AS usage_percentage_with_free_space, sum(t.monthly_actual_usage_cost) AS monthly_usage_cost
+   FROM ( SELECT t.dbase_name, t.schemanaame, t.tablename, t.megabytes, sum(t.megabytes)
```
Comment (neilsh) [inline src/AdminViews/v_cost_per_schema.sql]: s/t.schemanaame/t.schemaname

Comment (kerbelp) [inline src/AdminViews/v_cost_per_schema.sql]: Thanks for the comment. 
Fixed it and added comments and README.md update

**Comment by ericfe** [comment]
The view is very limited for a specific size/price cluster. Can you create a with clause that defines the number/type of nodes and prices so the future user can easily edit it for their use ? ( Number of nodes can be derived already and node type as well - by number of drives )

**Comment by chriz-bigdata** [comment]
This statement includes an example of how to use nodes, slices, disks to derive node type: https://github.com/awslabs/amazon-redshift-utils/blob/master/src/AdminScripts/unscanned_table_summary.sql

Could you also add clear placeholders for cost variables, with instructions/descriptions, rather than hard coded costs so that others can substitute values in?

**Comment by chriz-bigdata** [comment]
Closing due to inactivity.

**Pull request closed without merging**